### PR TITLE
Security updates

### PR DESCRIPTION
**Pull request by jbickar**
**Ready**

### CODE REVIEW

**Comment by jbickar** [comment]
Note: the update to `block_class` will come later, as it will require updates to the install profiles.

**Comment by jbickar** [comment]
@boznik I've added the `block_class` update to this PR, so it is now dependent on https://github.com/SU-SWS/stanford_install_tasks/pull/26.

File: sed-make.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Usage: ./sed-make <module> <oldversion> <newversion>
+# E.g., "./sed-make block_class 1.3 2.3"
+
+find . -name "*.make" -exec sed -i '' 's/'$1'\]\[version\] \= \"'$2'\"/'$1'\]\[version\] \= \"'$3'\"/g' "{}" \;
```
Comment (sherakama) [inline sed-make.sh:6]: Shea likes this 👍

Comment (jbickar) [inline sed-make.sh:6]: Bash ops

**Pull request merged**